event-sourcing/event-source-api/src/main/java/uk/gov/justice/services/eventsourcing/source/api/feed/event/EventEntryFeedMaxMinProviderProvider.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.services.eventsourcing.source.api.feed.event;
+
+import uk.gov.justice.services.eventsourcing.source.api.feed.common.FeedMaxMinProvider;
+
+import java.util.Comparator;
+import java.util.List;
+
+public class EventEntryFeedMaxMinProviderProvider implements FeedMaxMinProvider<EventEntry> {
+
+    final Comparator<EventEntry> sequenceComparator = (p1, p2) -> Long.compare(p1.getSequenceId(), p2.getSequenceId());
+
+    @Override
+    public long min(List<EventEntry> eventStreams) {
+        if (eventStreams.isEmpty()) {
```

Comment:
If you do min on am empty stream what do you get?

Reply:
Removed check for emptry stream as min returns 0 by default

Reply:
Not changed as your updating this class